visualizer/style.css

```diff
@@ -146,21 +151,80 @@ html * {
   display: none;
 }
 
+
+/* breakpoints helpers */
+
+.after-bp-1,
+.after-bp-2 {
+  display: none!important;
+}
+
+@media screen and (min-width: 630px) {
+  .after-bp-1 {
+    display: inherit!important;
+  }
+
+  .before-bp-1 {
+    display: none!important;
+  }
+}
+
+
+@media screen and (min-width: 930px) {
+  .after-bp-2 {
+    display: inherit!important;
+  }
+
+  .before-bp-2 {
+    display: none!important;
+  }
+}
+
 /* Header */
 
 .nc-header {
   flex-shrink: 0;
 }
 
+div#main-content {
+  display: flex;
+  flex-grow: 1;
+  flex-shrink: 1;
+  overflow: hidden;
+  position: relative;    
+  z-index: 0;
+}
+
+
 /* Footer */
 #footer {
   background-color: var(--opposite-contrast);
   color: var(--footer-color);
   flex-shrink: 0;
-  padding: 0 8px 6px;
+  padding: 0;
   border-top: 1px solid rgba(0, 0, 0, 0.5);
 }
 
+#footer #m-search-box-wrapper {
+  display: flex;
+  width: 100%;
+  padding: 10px;
+  background-color: var(--banner-bg-color);
+  border-bottom: 1px solid var(--light-glare);
+  display: none;
+}
+
+#footer #m-search-box-wrapper.show {
+  display: flex;
+}
```

Comment:
Would be nice to have consistent line spacing after closing curly braces. Something for the future, but we could easily add some formatting to CSS as an NPM script to enforce some simple styling rules 🙂

Reply:
agreed!